lib/ansible/plugins/callback/nrdp.py

```diff
@@ -0,0 +1,186 @@
+# -*- coding: utf-8 -*-
+# (c) 2018 Remi Verchere <remi@verchere.fr>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# Make coding more python3-ish
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    callback: nrdp
+    type: notification
+    author: "Remi VERCHERE (@rverchere)"
+    short_description: post task result to a nagios server through nrdp
+    description:
+        - this callback send playbook result to nagios
+        - nagios shall use NRDP to recive passive events
+        - the passive check is sent to a dedicated host/service for ansible
+    version_added: 2.7
```

Comment:
Update this to `2.8` because we're not in the 2.8 dev cycle, but other than that this looks good. Thanks!

Reply:
Done!